apps/server/src/modules/health/controller/health.controller.ts

```diff
@@ -0,0 +1,36 @@
+import { Controller, Get, HttpStatus, Res } from '@nestjs/common';
+import { Response } from 'express';
+
+import { HealthUC } from '../uc';
+import { HealthStatusResponse, HealthStatusResponseMapper } from './dto';
+
+@Controller('health')
+export class HealthController {
+	constructor(private readonly healthUc: HealthUC) {}
+
+	private readonly contentTypeApplicationHealthJSON = 'application/health+json';
+
+	@Get('self')
+	getSelfHealth(@Res({ passthrough: true }) res: Response): HealthStatusResponse {
+		res.contentType(this.contentTypeApplicationHealthJSON);
+
+		const healthStatus = this.healthUc.checkSelfHealth();
+
+		return HealthStatusResponseMapper.mapToResponse(healthStatus);
+	}
+
+	@Get()
+	async getOverallHealth(@Res({ passthrough: true }) res: Response): Promise<HealthStatusResponse> {
+		res.contentType(this.contentTypeApplicationHealthJSON);
+
+		const healthStatus = await this.healthUc.checkOverallHealth();
+
+		if (healthStatus.isPassed()) {
```

Comment:
Shorter version of if can be used

Reply:
You meant this kind of shorter if? https://github.com/hpi-schul-cloud/schulcloud-server/pull/4191/commits/5fc98c58fe786c71e1fa32f7729e1ec344814ef8

Reply:
Discussed with @sszafGCA offline, changed in https://github.com/hpi-schul-cloud/schulcloud-server/pull/4191/commits/5fc98c58fe786c71e1fa32f7729e1ec344814ef8 .